RubeesEat/Model/Person.cs

```diff
@@ -8,8 +8,9 @@ public class Person
     public Guid Id { get; }
     public string FirstName { get; }
     public string LastName { get; }
+    public string LoginName { get; }
 
-    public Person(Guid id, string firstName, string lastName)
+    public Person(Guid id, string firstName, string lastName, string loginName = null)
```

Comment:
We are also missing nullability. Please don't assign a default value as the default is not "User with no login name". Instead, users should have a login name and some don't because they are no longer active or whatever.

We also have the `Create` method above that does not have the added login name parameter.

Reply:
Since we are changing the function signature, we will have to change it for each call of the constructor or Create function (alltogether 63 usages, especially in tests). Is there a way to solve that other than manually adjusting each usage?

Reply:
Since we are changing the function signature, we will have to change it for each call of the constructor or Create function (alltogether 63 usages, especially in tests). Is there a way to solve that other than manually adjusting each usage?

Reply:
Never mind, I took care of it